src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,71 @@
-export const Tabs = () => {};
+import classNames from 'classnames';
+
+type Tab = {
+  id: string;
+  title: string;
+  content: string;
+};
+
+interface Props {
+  selectedTabId: string;
+  tabs: Tab[];
+  onTabSelected: (selectedTabId: string) => void;
+}
+
+export const Tabs: React.FC<Props> = ({
+  selectedTabId,
+  tabs,
+  onTabSelected,
+}) => {
+  const handleTabSelect = (tab: Tab) => {
+    if (tab.id === selectedTabId) {
+      return;
+    }
+
+    onTabSelected(tab.id);
+  };
+
+  const findTabById = (tabId: string): Tab | null => (
+    tabs.find(tab => tab.id === tabId) || null
+  );
+
+  return (
+    <div className="section">
+      <h1 className="title">
+        Selected tab is
+        {' '}
+        {findTabById(selectedTabId)?.title}
```

Comment:
move it to the variable
```suggestion
        {selectedTab.title}
```